yield_plugin/config/parameters.yaml

```diff
@@ -27,7 +27,7 @@ minimum_safety_gap_in_meters: 10.0
 # Maximum speed value to consider the ego vehicle stopped
 # Units: m/s
 # Value type: Desired
-max_stop_speed_in_ms: 1.0
+max_stop_speed_in_ms: 0.15
```

Comment:
Below this value, the vehicle forces the traj speed to 0m/s. So to command 0 from 1.0 m/s seemed pretty high. This is almost a pedestrian walking speed. Since it was in simulation the effect may not have felt as much. I got the new values from looking at the data and how it is printing towards the end.